Bio/PDB/Entity.py

```diff
@@ -13,20 +13,32 @@
 
 from collections import deque
 from copy import copy
+from typing import TYPE_CHECKING, Any, Dict, Generic, List, Optional, TypeVar, Union
```

Comment:
Repeating myself from a comment about Tuple, but likewise rather than importing Dict and List from typing, shouldn't we just used dict and list (lower case)?

Reply:
will do a big search-and-replace once we've dropped Python 3.8 :-)